### PR TITLE
Fix doc link

### PR DESCRIPTION
**Pull request by cijothomas**
Fixes #.

## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [ ] `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: docs/trace/extending-the-sdk/README.md [hunk 1]
```diff
@@ -172,16 +172,17 @@ but using the
...
-repo. The OpenTelemetry .NET Contrib Repo also has instrumentations for
-libraries like
-[ElasticSearch](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Contrib.Instrumentation.Elasticsearch),
-[EntityFramework](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Contrib.Instrumentation.EntityFrameworkCore)
+Some common examples of such libraries include
+[ASP.NET](../../../src/OpenTelemetry.Instrumentation.AspNet/README.md), [ASP.NET
+Core](../../../src/OpenTelemetry.Instrumentation.AspNetCore/README.md), [HTTP
+client .NET Core](../../../src/OpenTelemetry.Instrumentation.Http/README.md) .
+Instrumentation libraries for these are already provided in this repo. The
+OpenTelemetry .NET Contrib Repo also has instrumentations for libraries like
```
Comment (reyang) [inline docs/trace/extending-the-sdk/README.md:184]:
```suggestion
[OpenTelemetry .NET Contrib](https://github.com/open-telemetry/opentelemetry-dotnet-contrib) repository also has instrumentations for libraries like
```

**Review by reyang — APPROVED** [review]
LGTM.

File: docs/trace/extending-the-sdk/README.md [hunk 2]
```diff
@@ -172,16 +172,17 @@ but using the
 method. These are referred to as "legacy Activity" in this repo. These libraries
 already create activities but they do so by using the `Activity` constructor
 directly, rather than using `ActivitySource.StartActivity` method. These
-activities does not by default runs though the samplers, and will have their
+activities does not by default runs though the sampler, and will have their
```
Comment (utpilla) [inline docs/trace/extending-the-sdk/README.md:175]: nit: through* 
Could we rephrase it to this: "These activities by default do not run through samplers"?

**Review by utpilla — APPROVED** [review]

File: docs/trace/extending-the-sdk/README.md [hunk 3]
```diff
@@ -172,16 +172,19 @@ but using the
...
-[EntityFramework](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Contrib.Instrumentation.EntityFrameworkCore)
+Some common examples of such libraries include
+[ASP.NET](../../../src/OpenTelemetry.Instrumentation.AspNet/README.md), [ASP.NET
+Core](../../../src/OpenTelemetry.Instrumentation.AspNetCore/README.md), [HTTP
+client .NET Core](../../../src/OpenTelemetry.Instrumentation.Http/README.md) .
+Instrumentation libraries for these are already provided in this repo. The
+[OpenTelemetry .NET
+Contrib](https://github.com/open-telemetry/opentelemetry-dotnet-contrib)
+repostory also has instrumentations for libraries like
+[ElasticSearchClient](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Contrib.Instrumentation.ElasticsearchClient)
```
Comment (cijothomas) [inline docs/trace/extending-the-sdk/README.md:187]: Removed EntiryFramework from this list, as it was not falling in this category - it never created legacy activities :)
